### PR TITLE
Admin hotfix

### PR DESCRIPTION
**Pull request by tydolla00**

### CODE REVIEW

**Comment by tydolla00** [comment]
@Shargrove09 Make sure I didn't do anything consequential. 99% is ui changes so should be fine.

File: src/app/(routes)/admin/_components/MatchManager.tsx [hunk 1]
```diff
@@ -80,16 +72,26 @@ const MatchManager = (props: Props) => {
                   />
                 )}
               />
+              <div className="my-4 text-center text-lg">
+                Player Sessions for Match {matchIndex + 1}
+              </div>
+              {/* TODO Potentially move StatName here. Don't need to specify it for each control. */}
```
Comment (Shargrove09) [inline src/app/(routes)/admin/_components/MatchManager.tsx:78]: It's fine to move statName here. I just separated PlayerSession and PlayerStats for readability/understandability when I was thinking things through.  Just need to make sure that we keep track of PlayerSession for each stat for when we insert

Comment (tydolla00) [reply src/app/(routes)/admin/_components/MatchManager.tsx:78]: Yea once we figure out the player session stuff it'll make the UI more clear.

**Comment by tydolla00** [comment]
> image
> 
> @tydolla00 Thoughts on moving session name and/or thumbnail to be inline with session URL?

So the only reason I made it like that is because when you start resizing to smaller screens, I wanted the right side to be on top of the left (Session URL always on top of everything). And the only way to do what ur asking is to include session url in the grid layout. I could hack something together tho if it's really needed, just add -margin.

**Comment by shargrove-uft** [comment]
> o the only reason I made it like that is because when you start resizing to smaller screens, I wanted the right side to be on top of the left (Session URL always on top of everything). And the onl

I like the -margin option personally but agree that it'll probably be revisited after some usage so not super important

**Comment by Shargrove09** [comment]
> > image
> > @tydolla00 Thoughts on moving session name and/or thumbnail to be inline with session URL?
> 
> So the only reason I made it like that is because when you start resizing to smaller screens, I wanted the right side to be on top of the left (Session URL always on top of everything). And the only way to do what ur asking is to include session url in the grid layout. I could hack something together tho if it's really needed, just add -margin.
> 
> What do you think about this image
> 
> Using -margin. image
> 
> The problem with this tho is that the session url still spans two columns so it's just going to overlap the right side. I think once we have everything set in stone, we can brainstorm the flow.

Isn't session URL already apart of the grid? Regardless, I agree that it can be revisited later after some usage

File: src/app/(routes)/admin/_components/PlayerStatManager.tsx [hunk 2]
```diff
@@ -76,14 +77,14 @@ const PlayerStatManager = (props: Props) => {
...
+    <>
       {fields.map((field, index: number) => {
         return (
-          <div key={field.id} className="m-1 flex">
-            <span className="m-2 ml-0">{field.stat}</span>
-            <input
-              className="rounded-md p-2"
-              placeholder="Enter Stat"
+          <div key={field.id} className="my-4 flex gap-3">
+            {/* <span className="self-end">{field.stat}</span> */}
```
[the remark this answers is not in the document]
Comment (tydolla00) [reply src/app/(routes)/admin/_components/PlayerStatManager.tsx:84]: I think placeholder is better idea

[the remark this answers is not in the document]
Comment (Shargrove09) [reply src/app/(routes)/admin/_components/PlayerStatManager.tsx:84]: Down just worried about keeping track of which stat is which for games like CoD or RL after we input and are trying to check.

[the remark this answers is not in the document]
Comment (tydolla00) [reply src/app/(routes)/admin/_components/PlayerStatManager.tsx:84]: So this is something I wanted to discuss as well, do we want to add extra layers of validation to where they can't pick duplicate stats, matches per set must be a specific length, etc.

[the remark this answers is not in the document]
Comment (tydolla00) [reply src/app/(routes)/admin/_components/PlayerStatManager.tsx:91]: I'm going to come up with more ideas to better structure the UI. Closing this pr now, we can finish this discussion elsewhere.

**Pull request merged**